client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -1,23 +1,87 @@
+import { LaunchpadFlowTaskList } from './types';
+
 export const tasks = [
 	{
 		id: 'setup_newsletter',
 		isCompleted: true,
 		actionUrl: '#',
 		title: 'Set up Newsletter',
 		taskType: 'blog',
+		displayBadge: false,
+		badgeText: '',
 	},
 	{
 		id: 'plan_selected',
 		isCompleted: true,
 		taskType: 'blog',
 		actionUrl: '#',
 		title: 'Free Plan',
+		displayBadge: true,
+		badgeText: 'Personal',
 	},
 	{
 		id: 'subscribers_added',
-		isCompleted: false,
+		isCompleted: true,
```

Comment:
This is not a blocker for this PR, but a question: Do we know if this particular item can be skipped? If so, could isCompleted be false initially? And again if so, will we need to add some check for this?

Reply:
> Do we know if this particular item can be skipped? 

Yep! At least for the first iteration p1660839055880179/1660838121.081809-slack-CHN6J22MP